internal/plugin/connectors/tcp/mssql/connector.go

```diff
@@ -204,14 +242,12 @@ func (connector *SingleUseConnector) Connect(
 	// TODO: 	Replace this with an actual 'ok' message from the server
```

Comment:
These are existing comments (before your change), but can't these TODOs be combined (i.e. isn't the 'ok' message the same as the login response)?

Reply:
you're rght